src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,415 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.chatpageobject;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.gargoylesoftware.htmlunit.ElementNotFoundException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)